.gitignore

```diff
@@ -23,5 +23,10 @@ vendor/
 
 # Project specific ignore
 .env
+asset-body.yaml
+asset-body.json
```

Comment:
why we need this here?

Reply:
Was there for me to not push asset-body and discussion-body file to github.. Have removed it now.